ee/log/osquerylogs/log.go

```diff
@@ -64,9 +56,9 @@ func NewOsqueryLogAdapter(logger kitlog.Logger, rootDirectory string, opts ...Op
 func (l *OsqueryLogAdapter) Write(p []byte) (int, error) {
 	// Work around osquery being overly verbose with it's logs
 	// see: https://github.com/osquery/osquery/pull/6271
-	lf := l.levelFunc
+	level := l.level
 	if bytes.Contains(p, []byte("Executing scheduled query pack")) {
-		lf = level.Debug
+		level = slog.LevelDebug
```

Comment:
I think this is okay, but I could imagine some future work where we parsed the osquery log to determine the level.